frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareInstallerCard/InstallerDetailsWidget/InstallerDetailsWidget.tsx

```diff
@@ -88,10 +108,44 @@ const InstallerDetailsWidget = ({
         ""
       );
 
+    const renderSha256 = () => {
+      return sha256 ? (
+        <>
+          {" "}
+          &bull;{" "}
+          <span className={`${baseClass}__sha256`}>
+            <TooltipWrapper
+              tipContent={<>The software&apos;s SHA-256 hash.</>}
+              position="top"
+              showArrow
+              underline={false}
+            >
+              {sha256.slice(0, 6)}&hellip;
```

Comment:
nit: GitHub shows 7 chars of its hash, while Git shows 10. Not sure how we arrived at 6 here, but feels a _little_ weird. Not blocking though.